OrbitVulkanLayer/SubmissionTracker.h

```diff
@@ -471,6 +471,14 @@ class SubmissionTracker : public VulkanLayerProducer::CaptureStatusListener {
     std::vector<QueueSubmission> completed_submissions =
         PullCompletedSubmissions(device, query_pool);
 
+    // We sort the completed submissions by timestamp to avoid sending the capture events out of
+    // order.
+    std::sort(completed_submissions.begin(), completed_submissions.end(),
+              [](const QueueSubmission& left, const QueueSubmission& right) {
```

Comment:
Little note that common names for these are "lhs" and "rhs" (left-hand side, right-hand side), also with the advantage of being the same length.

Reply:
Done. Thanks.